docs/reference/aggregations/bucket/datehistogram-aggregation.asciidoc

```diff
@@ -113,13 +215,13 @@ Response:
 --------------------------------------------------
 // TESTRESPONSE[s/\.\.\./"took": $body.took,"timed_out": false,"_shards": $body._shards,"hits": $body.hits,/]
 
-==== Time Zone
+===== Time Zone
 
 Date-times are stored in Elasticsearch in UTC.  By default, all bucketing and
-rounding is also done in UTC. The `time_zone` parameter can be used to indicate
-that bucketing should use a different time zone.
+rounding is also done in UTC. Use the `time_zone` parameter to indicate
+that bucketing should use a different timezone.
 
-Time zones may either be specified as an ISO 8601 UTC offset (e.g. `+01:00` or
+You can specify timezones as either an ISO 8601 UTC offset (e.g. `+01:00` or
 `-08:00`)  or as a timezone id, an identifier used in the TZ database like
```

Comment:
Can we call this the _IANA timezone database_? There are other timezone databases with their own identification schemes (notably Windows has its own).

Reply:
done